vllm/v1/core/specialized_manager.py

```diff
@@ -1,30 +1,48 @@
 # SPDX-License-Identifier: Apache-2.0
+import sys
 from abc import ABC, abstractmethod
+from collections import defaultdict
+from collections.abc import Iterable
+from typing import Callable
+
+if sys.version_info >= (3, 12):
+    from typing import override
+else:
+    override = lambda x: x
```

Comment:
It seems `override` is unused in this file?

Reply:
I've added override to one function.